### ISSUE TITLE
Media server not being saved to .htaccess on 1.7.6.1

### ISSUE THREAD

**Issue by mrabsinthe** [PrestaShop/PrestaShop#16447]
**Describe the bug**
I already had a media server before upgrading to 1.7.6.1 from 1.7.5.2. When I changed the media server in BO/Performance, I noticed that is is not being saved to .htaccess file. You can see it correctly in BO but not in the .htaccess file.

**To Reproduce**
1. Go to 'BO/Performance'
2. Enter URL in Media server #1
3. Save
4. Download and check root .htaccess file

**Additional information**
PrestaShop version: 1.7.6.1
PHP version: 7.2.24

**Comment by khouloudbelguith** [comment]
Hi @mrabsinthe,

You need to follow these steps to regenerate the file .htaccess

- Rename the old file to .htaccess_old
- Disable friendly URL
- Enable friendly URL

Thanks to check and feedback

**Comment by mrabsinthe** [comment]
I know it works that way, but I believe that if you add media server url and save, you shouldn't be going to URLs, disable, save, enable and save. Why so many steps to just add media server, don't you think?

**Comment by khouloudbelguith** [comment]
@mrabsinthe, yes, I have the same issue with PS`1.7.5.0` & PS`1.7.6.1` & PS`1.7.6.2build1` & `develop` branch.
Ping @PrestaShop/product-team  what do you think?
I think it is a could improvement to add.
When we add a Media server, the .htaccess file should be automatically generated.
Or, just add a warning to regenerate .htaccess is required.

Thanks!

**Comment by hibatallahAouadni** [comment]
Ping @PrestaShop/prestashop-core-developers friendly reminder :wink: 
Could someone please answer @khouloudbelguith question :pray: 

Thanks in advance.

**Comment by Progi1984** [comment]
I think it's a @PrestaShop/product-team to decide to improve process.

**Comment by hibatallahAouadni** [comment]
Well then, thanks @Progi1984 :relaxed:
